### PR TITLE
274: Skeleton implementations of UpdateDeploy, UpdateTask

### PR DESCRIPTION
**Pull request by jparsons04**
**What does this pull request do?**
Small changes that starts to implement Update operations for Deploy and Task. It is not currently fully known what Update is supposed to do for these entities, but the operations need to be fleshed out to accept future implementation changes.

**Checklist**
- [ ] Unit tests
~- [ ] Smoke tests (if applicable)~
~- [ ] System tests (if applicable)~
~- [ ] Documentation (if applicable)~

### CODE REVIEW

**Comment by jparsons04** [comment]
It was requested that I base my work off the branch scheduler-skeleton and though the PR of issue #274 hasn't been merged into develop yet, the commit should be clear enough (I hope) about what I'm trying to do.

File: api/controllers/deploy.go [hunk 1]
```diff
@@ -78,3 +79,8 @@ func (d *DeployController) ListDeploys(c *fireball.Context) (fireball.Response,
 	return fireball.NewJSONResponse(200, summaries)
 
 }
+
+func (d *DeployController) UpdateDeploy(c *fireball.Context) (fireball.Response, error) {
```
Comment (zpatrick) [inline api/controllers/deploy.go]: There is no `UpdateDeploy`

File: api/controllers/task.go [hunk 2]
```diff
@@ -78,3 +79,8 @@ func (t *TaskController) ListTasks(c *fireball.Context) (fireball.Response, erro
 	return fireball.NewJSONResponse(200, summaries)
 
 }
+
+func (t *TaskController) UpdateTask(c *fireball.Context) (fireball.Response, error) {
```
Comment (zpatrick) [inline api/controllers/task.go]: There is no `UpdateTask`

File: api/controllers/deploy.go (hunk 1 above)
Comment (jparsons04) [reply api/controllers/deploy.go]: I have removed the implementation, including the route definition at the top of the file

File: api/controllers/task.go (hunk 2 above)
Comment (jparsons04) [reply api/controllers/task.go]: I have removed the implementation, including the route at the top of the file

**Comment by zpatrick** [comment]
closing since updates on these entities are not required.

**Pull request closed without merging**